src/Stripe.net/Services/BankAccounts/BankAccountService.cs

```diff
@@ -23,7 +23,7 @@ internal BankAccountService(ApiRequestor requestor)
         {
         }
 
-        [Obsolete("Please use the constructor that accepts an `ApiRequestor` instead.")]
+        [Obsolete("This member is deprecated. Access this service directly via the client with client.V1.BankAccounts")]
```

Comment:
We are not really referring to the StripeClient class here? Should this be

> Access this service directly via the StripeClient with client.V1.BankAccounts

or 

> Access this service directly via the client with StripeClient.V1.BankAccounts

Probably not the second coz it makes it look like a static method

Reply:
Also, are we sure we want to change this comment? From my understanding, this constructor was for folks who wanted to dig into the http internals, change a few things there and then run their requests. Isnt ApiRequestor is the right replacement in that case?

Reply:
https://github.com/stripe/stripe-dotnet/?tab=readme-ov-file#using-a-custom-httpclient

Reply:
> This member is deprecated

It is not really a "member" though